modules/appeals_api/app/swagger/appeals_api/v2/swagger_dev.json

```diff
@@ -1367,38 +1348,15 @@
                 "example": {
                   "errors": [
                     {
-                      "status": "404",
-                      "title": "Veteran not found",
-                      "detail": "A veteran with that SSN was not found in our systems."
```

Comment:
The original shape here was the reason I updated the HLR-specific contestable issues controller